README.md

```diff
@@ -33,6 +33,7 @@ The application contains the following important view controllers
 - **FormListViewController** - Loads and displays the available forms for authenticated users.
 - **OnePageFormViewController** -  Loads and displays a form on a single screen, providing validation before users submit forms.
 - **MultiPageFormViewController** - Loads and displays a form one field at a time. Uses the FieldViewController to display the fields.
+- **CaptureImageViewController** - Loads a form for taking pictures with device camera, upload an image from photo library and save it to S3.
```

Comment:
Perhaps: "Allows the user to take a picture or select one from the gallery, and uploads the image to AWS S3."